src/design-system/InputName.tsx

```diff
@@ -46,19 +41,18 @@ interface Props
   name?: string | undefined;
   persistChanges?: (changes: { name: string | undefined }) => void;
   setName: (name?: string) => void;
-  showIcon?: boolean;
+  border?: boolean;
 }
 
-const InputNameWithIcon: React.FC<Props> = ({
+const InputName: React.FC<Props> = ({
   name,
   persistChanges,
   setName,
-  showIcon,
+  border,
   ...passThruProps
 }) => {
   return (
-    <NameLabelDiv>
-      {showIcon && <IconFolder alt="folder" src={iconFolderSrc} />}
+    <NameLabelDiv border={border}>
```

Comment:
We no longer embed this Folder Icon within this otherwise generic component.  However we do pass in a new property `border` to optionally display a bottom border if the UI calls for it (i.e. in the `FacilityInputForm` in this case).